README.md

```diff
@@ -154,6 +154,8 @@ Options:
           HTTP request body.
   -D <BODY_PATH>
           HTTP request body from file.
+  -Z <BODY_PATH_LINES>
```

Comment:
I would feel better if all those conflicts/exclusions would be explained here (and in the actual help of course).